pyproject.toml

```diff
@@ -1,84 +1,19 @@
-[tool.flakeheaven]
-exclude = ["example.py", "setup.py", "README.rst"]
-max_line_length = 120
-show_source = true
-
-[tool.flakeheaven.plugins]
-pycodestyle = ["+*"]
-pyflakes = ["+*"]
-flake8-commas = ["+*"]
-flake8-quotes = ["+*"]
-pylint = ["+F*", "+E*", "-E0611", "-E1101", "-E0401", "-E1102", "-E1123"]
-
-# -- DEPHELL -- #
-
-[tool.dephell.main]
-from = {format = "flit", path = "pyproject.toml"}
-to = {format = "setuppy", path = "setup.py"}
-envs = ["main", "dev"]
-tag = "v."
-command = "python -m flakeheaven lint"
-
-[tool.dephell.pytest]
-from = {format = "flit", path = "pyproject.toml"}
-envs = ["main", "dev"]
-tests = ["tests"]
-command = "python -m pytest tests/"
-
-[tool.dephell.docs]
-from = {format = "flit", path = "pyproject.toml"}
-envs = ["docs"]
-command = "sphinx-build -W docs docs/build"
-
-[tool.dephell.isort]
-from = {format = "flit", path = "pyproject.toml"}
-envs = ["main", "dev"]
-command = "isort -rc -y"
-
-[tool.dephell.typing]
-from = {format = "flit", path = "pyproject.toml"}
-envs = ["main"]
-command = "mypy --ignore-missing-imports --allow-redefinition flakeheaven/"
-
-# -- ISORT -- #
-
-[tool.isort]
-line_length = 100
-combine_as_imports = true
-balanced_wrapping = true
-lines_after_imports = 2
-include_trailing_comma = true
-multi_line_output = 5
-
-import_heading_stdlib = "built-in"
-import_heading_thirdparty = "external"
-import_heading_firstparty = "project"
-import_heading_localfolder = "app"
-
-# -- FLIT -- #
-
-[tool.flit.metadata]
-module="flakeheaven"
-author="Gram (@orsinium)"
-author-email="master_fess@mail.ru"
-home-page="https://github.com/flakeheaven/flakeheaven"
-requires-python=">=3.5"
-requires=[
-    "colorama",
-    "entrypoints",
-    "flake8>=4.0.1",
-    "pygments",
-    "toml",
-    "urllib3",
-]
-description-file="README.md"
-classifiers=[
+[tool.poetry]
+name = "flakeheaven"
+version = "0.11.0"
+description = "FlakeHeaven is a [Flake8](https://gitlab.com/pycqa/flake8) wrapper to make it cool."
+authors = ["Gram <master_fess@mail.ru>"]
+repository = "https://github.com/flakeheaven/flakeheaven"
+homepage = "https://github.com/flakeheaven/flakeheaven"
+documentation = "https://flakeheaven.readthedocs.io/"
+readme = "README.md"
+license = "MIT"
+classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
     "Framework :: Flake8",
     "Intended Audience :: Developers",
     "License :: OSI Approved :: MIT License",
-    "Programming Language :: Python :: 3.5",
```

Comment:
doesn't seem like we actually support anymore? in fact we don't run tests for 3.6 either

Reply:
I seem to recall @pwoolvett saying that 3.6 tests are temporarily disabled.